src/charm.py

```diff
@@ -324,8 +324,6 @@ def _on_install(self, _event: InstallEvent) -> None:
         """
         self.unit.status = MaintenanceStatus("Installing packages")
 
-        self._update_kernel()
```

Comment:
The reconcile in the `_on_start` will still be executing this method.
The `_on_install` is taking too long, therefore removing this call as it will get executed again in `_on_start`.